third_party/amd/lib/TritonAMDGPUToLLVM/DotOpToLLVM/WMMA.cpp

```diff
@@ -339,7 +371,7 @@ LogicalResult convertDot(DotOp op, DotOpAdaptor adaptor,
   bool tied = numRepM % 2 == 0 && paddedOutputElemSize == 2;
   int tiedGroup = tied ? 2 : 1;
 
-  intrinsicName = addInstructionSuffix(intrinsicName, kWidth, aElemTy, bElemTy,
+  intrinsicName = addInstructionSuffix(intrinsicName, kBase, aElemTy, bElemTy,
```

Comment:
Instruction name should not use kBase. It should be kDim?

Reply:
the suffix here means the number of elements for a wave, so it should be kBase not kDim.  [for example](https://github.com/llvm/llvm-project/blob/30e9cbacab5b474de89992851f126fff300c1ab7/llvm/test/Analysis/UniformityAnalysis/AMDGPU/intrinsics.ll#L209):

```llvm
%tmp0 = call <8 x float> @llvm.amdgcn.wmma.f32.16x16x32.bf16.v8f32.v16bf16(i1 0, <16 x bfloat> %A, i1 0, <16 x bfloat> %B, i16 0, <8 x float> %C, i1 false, i1 false)
```

also I fixed the suffix for return type here (not sure why the original suffix can even be compiled in llvm...

Reply:
Now I see. For a single wmma instruction, kBase always equals kDim?